pandas/util/_str_methods.py

```diff
@@ -0,0 +1,18 @@
+"""
+Python3.9 introduces removesuffix and remove prefix.
+
+They're reimplemented here for use in Python3.8.
+"""
+from __future__ import annotations
+
+
+def removesuffix(string: str, suffix: str) -> str:
```

Comment:
Good one, thanks @twoertwein ! Even better, `pyupgrade` will automatically remove the outdated block when we run it with `--py39-plus` https://github.com/asottile/pyupgrade#python2-and-old-python3x-blocks

Reply:
Would have been okay with having this if/else within `removesuffix` to avoid having to change everything. When pyupgrade removes the lines within `removesuffix`, the PR author hopefully rewrites those calls to just `str.removesuffix(suffix`). Your solution is obviously more convenient for the future PR author :)

Reply:
OK I've added the `version_info` check in `_str_methods` too, so `pyupgrade --py39-plus` will rewrite them too and it should be clear to the committer / reviewer that the files can be removed